envelope.md

```diff
@@ -33,6 +34,8 @@ Base64() is [Base64 encoding](https://tools.ietf.org/html/rfc4648), transforming
 a byte sequence to a unicode string. Either standard or URL-safe encoding is
 allowed.
 
+PEM() is a [PEM encoding](https://datatracker.ietf.org/doc/html/rfc1421), transforming a DER (binary) encoded X.509 certificate to a base64 encoding with a one-line header and footer.
```

Comment:
nits:
- This is a certificate **chain**, so it is a series of PEM-encoded certificates, concatenated with newlines?
- wrap at 80 columns for consistency with rest of file